File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        int answer = 0;
        if(price >= 500000) answer = (int)(price * 0.8);
        else if(price >= 300000) answer = (int)(price * 0.9);
        else if(price >= 100000) answer = (int)(price * 0.95);
        else answer = price;
        return answer;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    answer = 0
    
    if (price >= 500000) :
        answer = int(price * 0.8)
    elif (price >= 300000) :
        answer = int(price * 0.9)
    elif (price >= 100000) :
        answer = int(price * 0.95)
    else :
        answer = price
    return answer